.travis.yml

```diff
@@ -1,7 +1,5 @@
 sudo: required
```

Comment:
@geek-soft this also isn't required anymore, it means a full vm is used rather than a docker container.